README.md

```diff
@@ -127,7 +127,20 @@ All you need is Docker and git (optional).
 
    Note: You can navigate to the repo url and download the repo as a zip file if you don't want to use git
 
-2. Build the docker image: `$ docker build -t puppet-oneview .`
+2. You can build docker image locally or pull the docker image from Docker Hub.
+   * Build the docker image locally: `$ docker build -t puppet-oneview .`
```

Comment:
Mention how to run the image locally also.

Reply:
Running of image is common for both local image and docker hub image  and it is part of section 3

Reply:
way of running would be same but not image names